src/app/profile/user-info/index.tsx

```diff
@@ -6,72 +6,125 @@ import Loading from "./loading";
-                THe internets's Friendliest Designer Kid.
-              </span>
-            </div>
+            <Link
+              className="flex  items-center justify-between rounded-3xl border-2 border-heliotrope bg-heliotrope px-6 py-3 transition duration-500 hover:scale-95"
+              href={"profile/edit"}
+            >
+              Edit
+            </Link>
           </div>
         </div>
-        <div className="mt-20 flex h-fit gap-20">
-          <Button
-            className="flex items-center justify-between rounded-3xl border-2 border-heliotrope px-7 py-4 transition duration-500 hover:scale-95"
-            fill={false}
-          >
-            Add Steam Account
-          </Button>
+        <div className="mt-10 flex w-full flex-col items-center border-t-2 border-mine-shaft-darken">
+          <div className="flex items-center justify-center gap-4 border-b-2 border-friar-gray px-24 py-4">
+            <h3 className="font-sans text-2xl font-semibold  ">Created</h3>
+            <span className="flex items-center rounded-3xl bg-friar-gray px-2 py-1">
+              3
+            </span>
+          </div>
 
-          <Link
-            className="flex  items-center justify-between rounded-3xl border-2 border-heliotrope bg-heliotrope px-6 py-3 transition duration-500 hover:scale-95"
-            href={"profile/edit"}
-          >
-            Edit
-          </Link>
+          <div className="mt-20 flex max-w-[1050px] flex-wrap  items-center gap-7">
+            {[
+              {
+                title: "Distant Galaxy",
+                image: "/images/category_placeholder.png",
+                authorName: "Animakid",
+                authorImage: "/images/avatar.png",
+              },
+              {
+                title: "Distant Galaxy",
+                image: "/images/category_placeholder.png",
+                authorName: "Animakid",
+                authorImage: "/images/avatar.png",
+              },
+              {
+                title: "Distant Galaxy",
+                image: "/images/category_placeholder.png",
+                authorName: "Animakid",
+                authorImage: "/images/avatar.png",
+              },
+              {
+                title: "Distant Galaxy",
+                image: "/images/category_placeholder.png",
+                authorName: "Animakid",
+                authorImage: "/images/avatar.png",
+              },
+              {
+                title: "Distant Galaxy",
+                image: "/images/category_placeholder.png",
+                authorName: "Animakid",
+                authorImage: "/images/avatar.png",
+              },
+            ].map((nft) => (
+              <NftCard {...nft} />
+            ))}
+
+            <Button
+              className="flex h-[434px] w-[330px] items-center justify-center  rounded-2xl border-2 border-heliotrope bg-mine-shaft-darken transition duration-500 hover:scale-95"
```

Comment:
I see that you added to some buttons `border-2 border-heliotrope`. You have this styles inside button component, but they don't work because you forget empty space.

<img width="378" alt="image" src="https://github.com/JackHamish/marketplace_front/assets/37477697/f3f37f27-7c54-48d9-88b7-35d32fb8e596">